### PR TITLE
TDP-3691 Updated documentation for http-voice-v2 transcription

### PR DESCRIPTION
**Pull request by muesono-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

File: voice/about.md [hunk 1]
```diff
@@ -5,9 +5,10 @@
 
 ## REST API Reference Index
 
-| Resource                           | Description                                                           |
-|:-----------------------------------|:----------------------------------------------------------------------|
-| [`/calls`](methods/calls/about.md) | The Calls resource lets you make phone calls and modify active calls. |
+| Resource                                     | Description                                                           |
+|:---------------------------------------------|:----------------------------------------------------------------------|
+| [`/calls`](methods/calls/about.md)           | The Calls resource lets you make phone calls and modify active calls. |
+| [`/recordings`](methods/recordings/about.md) | The Recordings resource lets you retrieve information about recordings created during calls as well as manage recordings on a live call. |
```
Comment (ghost) [inline voice/about.md:11]: to manage recordings on a live call you need to use /calls/id/recordings right?

File: voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+{% sample lang="http" %}
+
+```bash
+curl -X GET \
+    --url "https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recordings/{recordingId}/transcription" \
+    -u '{username}:{password}'
+```
+
+```
+{
```
Comment (ghost) [inline voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md:48]: do we need to document the output schema?

File: voice/about.md (hunk 1 above)
Comment (muesono-daitan) [reply voice/about.md:11]: That is true, but that endpoint is also present in the `recordings/about.md`.
There was a discussion about if that endpoint should be in the `calls/about.md` or in `recordings/about.md`. I didn't participate the discussion but right now this endpoint is present in both pages.

File: voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md (hunk 2 above)
Comment (muesono-daitan) [reply voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md]: I'm adding a description of the output we currently have, but please note that the output schema changes depending on the transcription options we use.
For instance, we don't use the channel identification feature or the speaker identification feature, so if we use that in the future, the schema would change.

Comment (muesono-daitan) [reply voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md]: Examples of how the schema changes with each feature:
channel identification: https://docs.aws.amazon.com/transcribe/latest/dg/how-channel-id.html
speaker identification: https://docs.aws.amazon.com/transcribe/latest/dg/how-diarization.html
alternative transcriptions: https://docs.aws.amazon.com/transcribe/latest/dg/how-alternatives.html

Comment (mmeyers-bw) [reply voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md]: those say that the account id and job id are part of the output schema.  Are we removing those?

Comment (muesono-daitan) [reply voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md]: Yes, the call-api removes that.
From the `com.bandwidth.voice.http.api.model.Transcription` object we take only the `results` property and that is what is sent to the customer.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]

File: voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+|:----------|:------------|
+| None      | None        |
+
+
+### Transcription Attributes
+
+<!--
+Description taken from:
+https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_Item.html
+https://docs.aws.amazon.com/transcribe/latest/dg/transcribe-dg.pdf
```
Comment (nashley-bw) [inline voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md]: Should we put this here? I'm fine with it, but earlier it sounded like we don't want to advertise our transcription provider.

File: voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdMedia.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+{% sample lang="http" %}
+
+```bash
+curl -X DELETE \
+    --url "https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recordings/{recordingId}/media" \
+     -u '{username}:{password}'
+```
+
+```
+HTTP/1.1 204
```
Comment (nashley-bw) [inline voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdMedia.md:36]: I think it's a great idea to include the response code here :+1: We should do this for the other endpoints too at some point.

File: voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md (hunk 3 above)
Comment (muesono-daitan) [reply voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md]: oops, I thought about leaving this here for our reference, but it does go as an html comment in the rendered doc. I'll remove it from here.
Thanks for noticing.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/startRecording.md [hunk 5]
```diff
@@ -9,15 +9,20 @@ All audio on both sides of the call will be recorded until the call ends or the
...
 | fileFormat                   | (optional) The audio format that the recording will be saved as: `mp3` or `wav`.  Default value is `wav`.                                                                                                              |
 | multiChannel                 | (optional) A boolean value indicating whether or not the recording file should separate each side of the call into its own audio channel. Default value is `false`. `true` results in two channels.                    |
 
 If the `recordingAvailableUrl` attribute is specified, then the [Recording Available](../callbacks/recordingAvailable.md)
 event is sent to the URL once the recording is available for download, indicating the `mediaUrl` and if there was any issue processing the recording.
 
+If the `transcriptionAvailableUrl` attribute is specified, then the [Transcription Available](../callbacks/transcriptionAvailable.md) event is sent to the URL once the transcription is available for download. BXML returned in response to this callback will be ignored.
+
 BXML returned in response to this callback will be ignored.
```
Comment (n-miles) [inline voice/bxml/verbs/startRecording.md:27]: This line should be with the above paragraph instead of this one

Comment (muesono-daitan) [reply voice/bxml/verbs/startRecording.md:27]: nice catch!

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Review by n-miles — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

File: voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+console.log(response.transcripts[0].confidence);
+```
+
+{% sample lang="php" %}
+
+```php
+// coming soon
+```
+
+### Example 2 of 2: Download a multi-channel transcription
```
Comment (wkersting-bw) [inline voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md:109]: This header "Example *2* of 2 ..." is displayed in the PHP section but none of the others.   They only have the header "Example *1* of 2 ..."

Comment (ghost) [reply voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md:109]: yeah that needs to be fixed.  @mhossomi

Comment (marcelohossomi) [reply voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md:109]: Nice catch, fixed it.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3691.s3-website-us-east-1.amazonaws.com/

**Pull request merged**